server/mailrepository/mailrepository-api/src/main/java/org/apache/james/mailrepository/api/properties/eventsourcing/EventSourcingMailRepositoryPropertiesStore.java

```diff
@@ -0,0 +1,72 @@
+/****************************************************************
```

Comment:
I hesitate to create new mailrepository-eventsourcing module, and If I place the impl in the mailrepository-cassandra module. Then it will be used only with cassandra-eventstore, incase of we have other middleware -eventstores, otherwise I have to import mailrepository-cassandra maven in other implementation and make poms become messy.

What would you suggest?

Reply:
or I can place it in an `impl` package, like what we did with JmapFiltering eventsourcing.